tiledb/sm/storage_manager/storage_manager.h

```diff
@@ -687,6 +687,19 @@ class StorageManager {
   Status get_latest_array_schema_uri(
       const URI& array_uri, URI* schema_uri) const;
 
+  /**
+   * Loads the schema of a schema uri from persistent storage into memory.
+   *
+   * @param array_schema_uri The URI path of the array.
+   * @param encryption_key The encryption key to use.
+   * @param array_schema The array schema to be retrieved.
+   * @return Status
+   */
+  Status load_array_schema_uri(
```

Comment:
Perhaps rename to `load_array_schema_from_uri`? Currently it is confusing, it seems that it is loading (discovering?) a URI instead of the actual schema.

Reply:
renamed to load_array_schema_from_uri. thanks for the comments